### DISCUSSION TITLE
血压增幅帖

### DISCUSSION THREAD

**Discussion by Tangri-Rat** [share121/inter-knot#459] [其他]
这个网站怎么打不开啊？我的网络也没问题啊？360为什么说这个网站有风险啊？vps是什么？从哪下载啊？​githu​b是什么？github加载太慢了啊。怎么搜索啊？怎么下载啊？我找不到下载按钮在哪怎么办？这么多文件夹我要下载哪一个啊？这也没有exe文件啊？下载了后下载到哪去了啊？怎么解压还要钱啊？要用什么软件免费解压啊？我重新下的解压软件为什么弹广告啊？解压出来的这一堆文件是什么啊？怎么启动啊？哪个是打开啊？打开了后要怎么办啊？机场是什么啊？订阅链接是什么啊？我注册完了怎么还要花钱买服务啊？免费订阅链接在哪啊？复制了链接在哪粘贴啊？导入链接后怎么没有用啊？更新订阅失败怎么办啊？节点怎么全部都ping不通啊？现在ping通了但是还是打不开网站啊？在哪修改代理模式啊？加载速度怎么那么慢啊？360还是提示网址危险不让访问啊？我怎么连百度都打不开了？现在打开了但为什么加载那么慢？大佬帮帮忙啊？我桌面上多了一个2345安全卫士？怎么设置代理规则啊？绕过中国大陆在哪啊？还是打不开创意工坊啊？steam怎么下载不了游戏了？速度才几十KB/s？油管网址是什么啊？打开为什么显示一片白啊？视频怎么加载不出来啊？谷歌能打开但为什么显示我在中国北京啊？为什么谷歌网址后面多了个.hk？为什么找不到黄网啊？为什么显示已启用安全搜索啊？怎么关闭啊？怎么还要登陆谷歌账号？​怎么注册谷歌账​号啊？一直卡在谷歌验证码怎么办啊？怎么我的手机号码接收不到验证码啊？大佬别走啊！​哥，你给的这链接怎么用?啥叫​磁力链​接啊?
下载完了，7Z是什么格式?解压完怎么是一堆文件夹?
把我桌面都搞乱了!不能全屏啊?没汉化?汉化包有吗?
为什么我打了汉化包变成汉字了?提示我按任意键进入游戏，任意键是哪个键?
问你呢怎么不说话?你给的资源我不找你问找谁问?​

**Comment by BSNHD** [comment]
写得很好，下次不要再写了，血压上来了[热]

**Comment by Mnpolize** [comment]
哪来的新任绳匠（恼

**Comment by lllliufeng** [comment]
推荐《零基础使用绳网：从装机到跑路》

**Comment by light814** [comment]
⊙﹏⊙∥

**Comment by XiaoFffeng** [comment]
不合格，建议重修

**Comment by capacitor1** [comment]
牛魔

**Comment by nahida0-o** [comment]
看得出来，写的人还是太有礼貌了，下次建议"你”起手，大佬更是不要提，每一句都能呛死人的那种才最好。

**Comment by ArgonTeam** [comment]
非常によく書かれています、次回は書かないでください

**Comment by figs23314** [comment]
合订本

**Comment by dengyunzhe** [comment]
这个委托，加钱也不接

**Comment by Thina-chng** [comment]
你吵到我的眼睛了

**Comment by 1ctrl-cv** [comment]
建议把“大佬”改成“你”
